### ISSUE TITLE
entropy in SIL policy loss

### ISSUE THREAD

**Issue by gabrieledcjr** [junhyukoh/self-imitation-learning#6]
In the equation in the paper, there is no entropy term in the SIL policy loss, how come in the code there is one?

self.loss = self.pg_loss - entropy * self.w_entropy
